package.json

```diff
@@ -30,7 +30,12 @@
     "build:umd:min": "cross-env BABEL_ENV=es NODE_ENV=production rollup -c -i src/index.js -o dist/redux-loop.min.js",
     "build": "yarn build:commonjs && yarn build:es && yarn build:umd && yarn build:umd:min",
     "prebuild": "yarn run clear",
-    "prepublishOnly": "yarn lint && yarn test && yarn build"
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "docs:clean": "rimraf _book",
+    "docs:prepare": "gitbook install",
+    "docs:build": "npm run docs:prepare && gitbook build -g redux-loo/redux-loop && cp logo/logo.png _book/gitbook/images/logo.png && cp logo/favicon.ico _book/gitbook/images",
```

Comment:
since redux-loop uses yarn, can you switch `npm run docs:prepare` to `yarn docs:prepare`  and all of the other npm run's to yarn too? Same with the other tasks.

also, looks like a type where it says `redux-loo/redux-loop` instead of `redux-loop/redux-loop` in a few spots.